development/image-builder/README.md

```diff
@@ -65,6 +65,45 @@ To use this feature, make sure that:
 * you have the `variants.yaml` file in the **same directory** as the `Dockerfile`
 * your `Dockerfile` contains `ARG` directives which are named after keys in `variants.yaml`
 
+## Image signing
+
+image-builder supports signing the images with pre-defined set of signing services to verify that image comes from trusted repository and has not been altered in the meantime.
+Every signing service can be enabled on repository and global levels.
+
+Example sign services configuration in config.yaml file:
+```yaml
+sign-config:
+  enabled-signers:
+    '*':
+      - default-signify
+    org/repo:
+      - repo-token-notary
+  signers:
+    - name: default-signify
+      type: notary
+      config:
+        endpoint: https://notary/sign
+        timeout: 5m
+        retry-timeout: 10s
+        secret:
+          path: /path/to/secret/file/signify.yaml
+          type: signify
+    - name: repo-token-notary
+      type: notary
+      config:
+        endpoint: https://repo-notary/sign
+        timeout: 5m
+        retry-timeout: 10s
+        secret:
+          path: /path/to/secret/file/token
+          type: token
+```
+
+All enabled signers under `'*'` will be used globally. Additionally, if repository contains another signer configuration in the `org/repo` key, image-builder will also use this service to sign image.
+When the job is running in CI (Prow), it will try to pick up the org/repo name from the default Prow variables. If binary is running outside of CI, `--repo` flag will have to be used. Otherwise, the configuration will not be used.
```

Comment:
Not sure if "org/repo name" here means the same as "`org/repo` key" in the previous sentence, or whether it's something else/new.
Depending on that, I have two different suggestions:

```suggestion
If the job is running in CI (Prow), it picks up the org or repo name from the default Prow variables.
If the binary is running outside of CI, you must set the `--repo` flag. Otherwise, the configuration will not be used.
```

or

```suggestion
If the job is running in CI (Prow), it picks up the `org/repo` key from the default Prow variables.
If the binary is running outside of CI, you must set the `--repo` flag. Otherwise, the configuration will not be used.
```

Reply:
@NHingerl it's not a single key. It's a set of 2 env variables: `REPO_OWNER` and `REPO_NAME`. IF program works in CI, then it builds `org/repo` value from those 2 variables.

Reply:
Then does one of my suggestions above fit, or are both not good?
Background: I want to get rid of the `/`, unless that is in the name of the key (like I assume it is in line 102).